clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2860,9 +2861,25 @@ static void setLinkageForGV(llvm::GlobalValue *GV, const NamedDecl *ND) {
     GV->setLinkage(llvm::GlobalValue::ExternalWeakLinkage);
 }
 
+static bool HasExistingGeneralizedTypeMD(llvm::Function *F) {
```

Comment:
```suggestion
static bool hasExistingGeneralizedTypeMD(llvm::Function *F) {
```
https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly